### PR TITLE
ToS for video and sip

### PR DESCRIPTION
**Pull request by cspiel1**
Allows to configure ToS differently for RTP video/audio and SIP

Was not sure which default value we should use for SIP.

### CODE REVIEW

File: docs/examples/config [hunk 1]
```diff
@@ -47,6 +48,7 @@ videnc_format		yuv420p
 
 # AVT - Audio/Video Transport
 rtp_tos			184
+rtpv_tos		104
```
Comment (alfredh) [inline docs/examples/config:51]: perhaps `v` should be written as `video` instead ? it is easier to understand

File: docs/examples/config [hunk 2]
```diff
@@ -13,6 +13,7 @@ poll_method		epoll		# poll, select, epoll ..
 sip_cafile		/etc/ssl/certs/ca-certificates.crt
 #sip_trans_def		udp
 sip_verify_server	yes
+sip_tos			104
```
Comment (alfredh) [inline docs/examples/config:16]: is TOS for SIP really needed ?

TOS/QoS is mainly about the real-time media traffic that gets higher priority in a network.

**Comment by alfredh** [comment]
perhaps the TOS field bit-fields can be expanded and documented in the sample config ?

**Comment by alfredh** [comment]
@sreimers can you please merge the re patch ?

File: docs/examples/config (hunk 2 above)
Comment (cspiel1) [reply docs/examples/config:16]: I think so. It seems that different manufacturer also set it for SIP with different default values. https://kb.smartvox.co.uk/ip-phones/sip-configuration/voip-qos-settings-part-2/

We need it also for our products.

**Comment by cspiel1** [comment]
Added a commit for TOS field explanation. Tried to understand the values for "precedence" field. The answer seems not to be simple enough to put it there. A lot of RFC upgrades. Useful values for the whole ToS byte are listed here: https://linuxreviews.org/Type_of_Service_(ToS)_and_DSCP_Values

**Comment by cspiel1** [comment]
rebased to master

**Pull request merged**